pkg/cloudprovider/plugins/aws/aws_api_wrappers.go

```diff
@@ -144,3 +150,8 @@ func (ec2Wrapper *awsEC2WrapperImpl) describeVpcPeeringConnectionsWrapper(input
 	*ec2.DescribeVpcPeeringConnectionsOutput, error) {
 	return ec2Wrapper.ec2.DescribeVpcPeeringConnections(input)
 }
+
+// getRegion returns the region of the ec2 client.
+func (ec2Wrapper *awsEC2WrapperImpl) getRegion() string {
+	return *ec2Wrapper.ec2.Config.Region
```

Comment:
Since you are dereferencing the pointer, check if there id any scenario when region could be empty.

Reply:
I'll add a check, but for AWS clients, the region is a required field.